### DISCUSSION TITLE
留言板功能

### DISCUSSION THREAD

**Discussion by chuxinyuan** [CaiJimmy/hugo-theme-stack#908] [Q&A]
我在项目的 README 里看到了留言板功能菜单，可是部署后却没有，不知道为什么？
另外，我发现了神奇的留言板效果，详见：<https://tctcab.top/#contact>
请问下如何实现？

**Comment by tcgriffith** [comment]
> 我在项目的 README 里看到了留言板功能菜单，可是部署后却没有，不知道为什么？ 另外，我发现了神奇的留言板效果，详见：https://tctcab.top/#contact 请问下如何实现？

想知道那我整理一下代码 ;)

**Reply by chuxinyuan** [reply]
谢谢先！这个一定花了很多时间才搞定的，直接伸手要感觉不妥，当时我没好意思开口:smile:

**Reply by tcgriffith** [reply]
不复杂， 没啥不妥的哈哈 都是开源的东东 有用就拿去玩
